src/gwt/src/org/rstudio/core/rebind/JavaScriptPassthroughGenerator.java

```diff
@@ -138,7 +138,7 @@ private void printParams(SourceWriter w,
                   }
                   else if (simpleName.equals("CommandWithArg"))
                   {
-                     w.print("(Ljava/lang/Object;)(result)");
+                     w.print("(*)(result)");
```

Comment:
I discovered that GWT uses * as a wildcard placeholder for types; effectively meaning "infer and box the underlying JS object as appropriate".

Reply:
That's very cool!